src/commands/add.js

```diff
@@ -1,16 +1,56 @@
 // @flow
+import Project from '../Project';
+import Package from '../Package';
 import * as options from '../utils/options';
-import { BoltError } from '../utils/errors';
+import * as logger from '../utils/logger';
+import addDependenciesToPackage from '../utils/addDependenciesToPackages';
+import type { Dependency, configDependencyType } from '../types';
 
-export type AddOptions = {};
+export type AddOptions = {
+  cwd?: string,
+  deps: Array<Dependency>,
+  type: configDependencyType
+};
+
+const depTypeFlags = {
```

Comment:
This and the toWorkspaceAddOptions are almost identical to the ones in add() bar the workspace arg. I'd love to dedupe if possible...